### DISCUSSION TITLE
contributions

### DISCUSSION THREAD

**Discussion by Leo3213** [Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3#6] [Show and tell]
If anyone has the same laptop had has fixed something that isnt inclued add it here so eveyone can use it :)

**Comment by robi62** [comment]
Here are the kext for the card reader Try it
[Cardreader.zip](https://github.com/Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3/files/7366424/Cardreader.zip)
